packages/esbuild-transpiler/package.json

```diff
@@ -2,32 +2,48 @@
   "name": "@hono/esbuild-transpiler",
   "version": "0.1.3",
   "description": "esbuild Transpiler Middleware for Hono",
-  "main": "dist/index.js",
   "type": "module",
+  "module": "dist/index.js",
   "types": "dist/index.d.ts",
   "files": [
     "dist"
   ],
   "scripts": {
-    "test": "vitest run",
-    "build": "tsup ./src/*.ts ./src/transpilers/*.ts --format esm,cjs --dts --no-splitting --external esbuild-wasm,esbuild",
-    "publint": "publint",
-    "release": "yarn build && yarn test && yarn publint && yarn publish"
+    "build": "tsup ./src/*.ts ./src/transpilers/*.ts --no-splitting --external esbuild-wasm,esbuild",
```

Comment:
It looks like `--no-splitting` is actually doing something in this case, so I've left it in